sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -4297,7 +4297,9 @@ var setParentClass = function(child, parent) {
     };
 
     glsBuiltinPrecisionTests.Radians.prototype.doExpand = function(ctx, args) {
-        var val = new glsBuiltinPrecisionTests.Constant(Math.PI / 180);
+        var val = app(new glsBuiltinPrecisionTests.Div(),
+                                                      new glsBuiltinPrecisionTests.Constant(Math.PI),
+                                                      new glsBuiltinPrecisionTests.Constant(180));
         return new glsBuiltinPrecisionTests.Apply('float',
                                                   new glsBuiltinPrecisionTests.Mul(),
                                                   val,
```

Comment:
See the native code: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/glshared/glsBuiltinPrecisionTests.cpp#2235. It will use Div() operation, not a constant value. The Div() operation will bring in precision calculation, see https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/glshared/glsBuiltinPrecisionTests.cpp#2047. 

If we use a constant value like the original JS implementation, the precision loss caused by the division of PI / 180.0 will not be considered. So the final precision range is very small, making the tests failed. However, the output from radians() in shader in WebGL is exactly the same with that in native. And the result is thought to be acceptable in native deqp. The only reason is the precision ranges are different.